integrations/github/src/haystack_integrations/tools/github/issue_commenter_tool.py

```diff
@@ -1,19 +1,46 @@
 # SPDX-FileCopyrightText: 2023-present deepset GmbH <info@deepset.ai>
 #
 # SPDX-License-Identifier: Apache-2.0
-from typing import Any, Dict, Optional
+from typing import Any, Callable, Dict, Optional, Union
 
-from haystack import default_from_dict, default_to_dict
 from haystack.tools import ComponentTool
 from haystack.utils import Secret, deserialize_secrets_inplace
 
 from haystack_integrations.components.connectors.github.issue_commenter import GitHubIssueCommenter
 from haystack_integrations.prompts.github.issue_commenter_prompt import ISSUE_COMMENTER_PROMPT, ISSUE_COMMENTER_SCHEMA
+from haystack_integrations.tools.github.utils import deserialize_handlers, serialize_handlers
 
 
 class GitHubIssueCommenterTool(ComponentTool):
     """
     A tool for commenting on GitHub issues.
+
+    :param name: Optional name for the tool.
+    :param description: Optional description.
```

Comment:
I do realize this is a bit confusing in our tools, but it seems that if we define a `__init__` then these docstrings are put under the `__init__` def. If there is no `__init__` defined like in `Tool` then we put it in the class description.

Reply:
Ah, of course! Do you think we should a usage example here then (in addition to moving the param docstrings to the init)? I realized that's missing too.